### ISSUE TITLE
Real IP when behind a proxy.

### ISSUE THREAD

**Issue by adamlc** [sameersbn/docker-gitlab#884]
Not really an issue as such, but I've just come across a problem with rack attack blocking my nginx proxy container's IP address. To get round it for now I've just disabled it. I noticed inside gitlab that its not picking IP addresses up properly, instead its using the proxy's IP.

I'm using jwilder/nginx-proxy to proxy gitlab. Are there any proxy headers or anything I should change to get gitlab to pick up proper IP addresses? Is so it might be worth updating the docs as its probably quite a common setup :)

**Comment by SolidNerd** [comment]
Hey, 
sorry for the delay. This depends on your docker setup. I reckon that you don't receive the correct IP Address within the nginx-proxy container. This could be solved on several ways. There is in issue for that problem at jwilder/nginx-proxy repo. https://github.com/jwilder/nginx-proxy/issues/130 . There you should found more information. If you need more help feel free to ask.

**Comment by reynold-lariza** [comment]
Hey guys, just want to re-open this. I used the basic nginx container with `--net=host`

I also followed for the AWS ELB:

https://easyengine.io/tutorials/nginx/amazon-elastic-load-balancer-forward-real-ip/

Now I'm able to verify that the nginx is able to capture the actual client IP address, when I enable ` proxy_set_header  X-Forwarded-Host $remote_addr;`

However, how do I pass this IP to gitlab? cause what it currently gets is the private IP address of the docker host where nginx proxy is running.

Anyone encountered the same issue, and how were you able to solve this issue?

Thanks,
